oci/utils/tarball.go

```diff
@@ -0,0 +1,41 @@
+package utils
+
+import (
+	"archive/tar"
+	"fmt"
+	"io"
+	"io/fs"
+	"path/filepath"
+)
+
+// OpenFromTarOCILayer returns a reader for the given file from the provided
+// tarball reader.
+func OpenFromTarOCILayer(r io.Reader, path string) (io.Reader, error) {
+	tarPath, err := filepath.Rel("/", path)
+	if err != nil {
+		return nil, fmt.Errorf("trimming leading separator from path %q: %w", path, err)
```

Comment:
We usually format these as `could not ...`, so let's just do the same:

```suggestion
		return nil, fmt.Errorf("could not trim leading separator from path %q: %w", path, err)
```